setup.py

```diff
@@ -112,6 +112,7 @@
                                 op.join('data', 'FreeSurferColorLUT.txt'),
                                 op.join('data', 'image', '*gif'),
                                 op.join('data', 'image', '*lout'),
+                                op.join('data', 'fsaverage', '*.fif'),
```

Comment:
there is a change in `manifest.in`, it's the first line of changes in the comparison...?